overlord/install/install.go

```diff
@@ -81,13 +82,62 @@ type EncryptionSupportInfo struct {
 	UnavailableWarning string
 }
 
+// CompSeedInfo contains information for a component from the seed and
+// from its metadata.
+type CompSeedInfo struct {
+	CompInfo *snap.ComponentInfo
+	CompSeed *seed.Component
```

Comment:
Would read even nicer when used? Anyway, nitpicks here, feel free to ignore
```suggestion
	Info *snap.ComponentInfo
	Seed *seed.Component
```

Reply:
Changed